x-pack/plugins/observability/public/pages/alert_details/components/page_title.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiBadge, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export interface PageTitleProps {
+  title: string | undefined;
+  active?: boolean;
+}
+
+export function PageTitle({ title, active }: PageTitleProps) {
+  const label = active
+    ? i18n.translate('xpack.observability.alertDetails.alertActiveState', {
+        defaultMessage: 'Active',
+      })
+    : i18n.translate('xpack.observability.alertDetails.alertRecoveredState', {
+        defaultMessage: 'Recovered',
+      });
+
+  return (
+    <div data-test-subj="page-title-container">
+      {title}
+
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <div>
+            {typeof active === 'boolean' ? (
```

Comment:
Why do we need to check typeof active here?

Reply:
Don’t pass the prop, no badge gets rendered. 
Pass the prop with ‘false’, badge gets rendered with label ‘Recovered’
Pass the prop with ‘true’ badge gets rendered with label ‘Active’